datashader/tests/test_pandas.py

```diff
@@ -25,11 +25,19 @@
                       'plusminus': np.arange(20, dtype='f8')*([1, -1]*10),
                       'empty_bin': np.array([0.] * 15 + [np.nan] * 5),
                       'cat': ['a']*5 + ['b']*5 + ['c']*5 + ['d']*5,
+                      'cat2': ['a', 'b', 'c', 'd']*5,
                       'cat_int': np.array([10]*5 + [11]*5 + [12]*5 + [13]*5)})
 df_pd.cat = df_pd.cat.astype('category')
+df_pd.cat2 = df_pd.cat2.astype('category')
 df_pd.at[2,'f32'] = nan
 df_pd.at[2,'f64'] = nan
 df_pd.at[2,'plusminus'] = nan
+# x          0  0   0  0 0   0 0  0 0  0   1   1  1   1  1    1  1   1  1   1
```

Comment:
I find myself creating this manual table whenever I need to check new tests, so here including it to make it easier to check new tests in future.